xdp-loader/xdp-loader.c

```diff
@@ -67,6 +69,13 @@ static struct prog_option load_options[] = {
 		      .metavar = "<filenames>",
 		      .required = true,
 		      .help = "Load programs from <filenames>"),
+	DEFINE_OPTION("prio", OPT_U32, struct loadopt, prio,
+		      .short_opt = 'P',
+		      .help = "Set run priority of program"),
+	DEFINE_OPTION("actions", OPT_STRING, struct loadopt, actions,
+		      .metavar = "<actions>",
+		      .short_opt = 'A',
+		      .help = "Chain call actions (default: XDP_PASS). e.g. XDP_PASS,XDP_DROP"),
```

Comment:
Make this an OPT_FLAGS instead and you don't need the `parse_chain_call_actions()` function (the option parsing will handle that).

Reply:
Yes! The OPT_FLAGS is the right choice, I will use that.